### PR TITLE
Fix /status for UDMP

### LINKED ISSUE #16: /status needs proxied to /proxy/network
/status does not work on UDM Pro but does work when accessed via /proxy/network/status

Controller version `5.14.22`
Build `atag_5.14.22_13865`)

### PR DESCRIPTION
**Pull request by solarmicrobe**
/status needs redirected to /proxy/network/status

The least amount of code change seemed to be splitting the path into a root path and API path.

Resolves #16

### CODE REVIEW

**Comment by paultyng** [comment]
Thanks, this looks reasonable but hard to test of course. I really need to just get a UDM pro.

Not sure why tests didn't run, will need to poke at the repo.

**Comment by paultyng** [comment]
This seemed to fail on non-UDMP for status checks (in this case a docker container version of the controller). Will need to investigate alternatives.

**Comment by paultyng** [comment]
@solarmicrobe I pushed a new commit that modifies this slightly, only modifying the calculation of the status request URL. This passes the provider tests on non-UDMP environments, but not sure if it still solves the UDMP issue.

File: unifi/unifi.go [hunk 1]
```diff
@@ -114,16 +114,16 @@ func (c *Client) setAPIUrlStyle(ctx context.Context) error {
 
 	if resp.StatusCode == http.StatusOK {
 		// the new API returns a 200 for a / request
-		c.apiRoot = apiRootNew
 		c.apiPath = apiPathNew
 		c.loginPath = loginPathNew
+		c.loginPath = statusPathNew
```
Comment (solarmicrobe) [inline unifi/unifi.go]: Type, fixed this in the new commit

**Comment by solarmicrobe** [comment]
Newest commit works for me. The provider tests weren't working for me but running this version by hand (and the only change being in the UDM/P path) seems to look good.

**Review by paultyng — APPROVED** [review]

**Pull request merged**